docs/Telemetry.md

```diff
@@ -308,6 +308,14 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 | Autocomplete URL tip tapped | action   | click | tip | add_autocomplete_url_tip    |        |
 | Homescreen tips enabled/disabled | action   | click | tip | add_to_homescreen_tip    |        |
 
+### Homescreen Tips
+| Event               | category | method | object         | extra         |
+|---------------------|----------|--------|----------------|---------------|
+| Open Crash Reporter | action   | show   | crash_reporter |               |
+| Close Tab           | action   | click  | crash_reporter | submit_crash  |
```

Comment:
Alternatively you could just write true/false instead of submit_crash - the name of the value doesn't matter, just all the possible values themselves.